packages/shared/lib/core/layer-2/utils/parseLayer2MetadataForTransferV2.ts

```diff
@@ -4,18 +4,15 @@ import { CONTRACT_FUNCTIONS, TARGET_CONTRACTS } from '../constants'
 import { Allowance } from '../enums'
 import { ReadSpecialStream } from '../classes'
 
-// Function to parse data from the current metadata, using the new encoding where the shimmer chainId is 1072
+// Function to parse data from the current metadata, using the new encoding where the shimmer chainId is 1073
 export function parseLayer2MetadataForTransferV2(metadata: Uint8Array): ILayer2TransferAllowanceMetadata {
```

Comment:
This encoding is not v2 anymore but rather v3, but in order to avoid maintaining legacy code forever, i would remove this and just have support with the latest version